R/upkeep.R

```diff
@@ -16,13 +16,13 @@
 #' @export
 #' @examples
 #' \dontrun{
-#' use_upkeep_issue(2023)
+#' use_upkeep_issue()
 #' }
 use_upkeep_issue <- function(year = NULL) {
   make_upkeep_issue(year = year, tidy = FALSE)
 }
 
-make_upkeep_issue <- function(year, tidy) {
+make_upkeep_issue <- function(year, last_year, tidy) {
```

Comment:
I wonder if it's better to frame this as "previous (upkeep) year"? Seems less vulnerable to confusion about what "last year" actually means. So I'm proposing `previous_year` or `prev_year`.

Reply:
Also, from #1916, do you think we should be incorporating this idea throughout?

> Recording year and month is probably worthwhile.

Reply:
I wondered about this as well. On one hand we are currently not using month to anything (which is we I went the way I did), but on the other hand we may do upkeep biannually which means recording year is not enough.

I'm happy to add month as well and then we can always figure out if/how we want to use that information